zilencer/models.py

```diff
@@ -207,11 +207,10 @@ class Meta:
                 condition=Q(subgroup__isnull=True),
                 name="unique_remote_installation_count_null_subgroup",
             ),
-        ]
-        indexes = [
-            models.Index(
+            UniqueConstraint(
                 fields=["server", "remote_id"],
-                name="zilencer_remoteinstallat_server_id_remote_id_f72e4c30_idx",
+                condition=Q(remote_id__isnull=False),
```

Comment:
What's the story behind this `condition`? Might be worth an explanatory comment for posterity.